airflow/jobs.py

```diff
@@ -592,6 +592,10 @@ def manage_slas(self, dag, session=None):
             dttm = ti.execution_date
             if task.sla:
                 dttm = dag.following_schedule(dttm)
+                # Skip tasks for @once dags
+                if not dttm and dag.schedule_interval == '@once':
+                    self.logger.warn(' --------------> SLAs not supported in @once DAGs')
```

Comment:
Thanks Sid. Is this documented somewhere? 

We would love to have SLAs for `@once` DAGs.

Reply:
@Tagar I'm going to fix this issue during next week. I will fix the root cause of this issue. 
I don't agree with the solution proposed in this pull request.

Reply:
good idea. thanks @mido3